snuba/schemas.py

```diff
@@ -243,6 +247,7 @@ def get_time_series_extension_properties(default_granularity: int, default_windo
             'granularity': {
                 'type': 'number',
                 'default': default_granularity,
+                'minimum': 0,
```

Comment:
This is correct for the type, but would actually lead to a divide by zero if used in practice. Might be more appropriate to make this a minimum of 1?

Reply:
Same. Then if one day someone will come up with a reasonable use case (probably not) we can release the constraint

Reply:
Also updated this with 152c7a76510a7c167089dc4c3b3e1b1e74d901af. I guess technically this could be `exclusiveMinimum` of `0` but there's no point to doing subsecond granularity since the `DateTime` is only precise down to seconds (at least with this dataset) so this should be fine. We might want to change this to be an integer later since there's no benefit to using a float (I suspect it would likely error on the ClickHouse side if provided a float, but haven't checked.)